lib/tty/logger/handlers/console.rb

```diff
@@ -148,8 +150,8 @@ def format_backtrace(event)
         #
         # @api private
         def configure_styles(event)
-          style = STYLES.fetch(event.metadata[:name].to_sym, {}).dup
-          (@styles[event.metadata[:name].to_sym] || {}).each do |k, v|
+          style = STYLES.fetch(event.metadata[:name]&.to_sym, {}).dup
+          (@styles[event.metadata[:name]&.to_sym] || {}).each do |k, v|
```

Comment:
Unfortunately, we can't have nice things. This gem needs to work with older Rubies, for now, I'm afraid all above `2.0`. Can this be changed to a guard clause?

Reply:
Done! Please have a look at the fix and let me know if it's OK for you.